src/surfaces/actionBar/actionBarSearchSelect.tsx

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import Select from '../../inputs/select';
+import Divider from '../../dataDisplay/divider';
+import {
+    UI_CLASS_NAME,
+} from '../../global/constants';
+
+type SelectOption<TValue = number, TLabel = string> = {
+    label: TLabel;
+    value: TValue;
+};
```

Comment:
NAB: Should we move this type declaration into its own file, so it is shareable more broadly?

Reply:
Completed